### PR TITLE
Bugfix/abi breaking changes

### PR DESCRIPTION
**Pull request by wmaxey**
I've merged #165 and #161 as these are the ABI breaking changes we're aiming to get in this release.

### CODE REVIEW

**Comment by wmaxey** [comment]
VDVS: http://scdvs.nvidia.com/Build_Results?virtualId=1000073819&which_page=current_build

**Comment by wmaxey** [comment]
VDVS: https://builds4u.nvidia.com/dvs/#/change/3023296555722932.1?eventType=Virtual

File: libcxx/include/chrono [hunk 1]
```diff
@@ -3232,9 +3237,9 @@ inline namespace literals
     }
 
     _LIBCUDACXX_INLINE_VISIBILITY
-    constexpr chrono::duration<long double> operator""s(long double __s)
+    constexpr chrono::duration<double> operator""s(long double __s)
     {
-        return chrono::duration<long double> (__s);
+        return chrono::duration<double> (__s);
```
Comment (griwes) [inline libcxx/include/chrono:3242]: Should be `_LIBCUDACXX_CHRONO_LITERAL_INTERNAL_T` too, not `double`.

**Review by griwes — APPROVED** [review]
Looks good to me once the one comment is fixed.

**Pull request merged**